index.js

```diff
@@ -15,9 +15,11 @@ const defaults = require('./lib/defaults');
 const logger = require('./lib/logger');
 const helmet = require('helmet');
 
+const customConfig = {};
+
 const getConfig = function getConfig() {
   const args = [].slice.call(arguments);
-  return Object.assign.apply(null, [{}, defaults].concat(args));
+  return Object.assign.apply(null, [{}, defaults, customConfig].concat(args));
```

Comment:
I'd personally remove the case where this function can be called with more than one argument (namely when passing additional config to `start`), and hence be able to clean up this code, but it's out of scope of this PR.